juniorguru_chick/bot.py

```diff
@@ -30,6 +30,14 @@ async def on_message(message: discord.Message) -> None:
         logger.info("Message sent by the bot itself, skipping")
         return
     if message.guild is None:
+        try:
+            response = "Píp píp píp! Jsem jen malé kuřátko, které neumí číst soukromé zprávy a odpovídat na ně.\
+                Tvou zprávu si nikdo nepřečte. Pokud se chceš na něco zeptat, zkus kanál <#769966886598737931>\
+                nebo napiš do soukromé zprávy komukoliv z moderátorů. Rádi tě nasměrují."
+            await message.channel.send(response)
+        # Skip responding if the user has private DM's disabled
+        except discord.errors.Forbidden:
+            pass
```

Comment:
We could move the comment to the except block:
```suggestion
            logger.info("Skipping responding, the user has private DM's disabled")
```